app/views/calculation/gain/valueBeforeLegislationStart.scala.html

```diff
@@ -40,7 +40,9 @@
 ) {
     @errorSummary(valueBeforeLegislationStartForm.errors)
 
-    <h1 class="govuk-heading-xl">@messages("calc.resident.shares.valueBeforeLegislationStart.question")</h1>
+    <div style="width: 510px">
```

Comment:
I'm not sure on this 🤔 
it wouldn't be responsive - so like when the page is resized for different screen sizes might be good to look into responsive sizes like percentages or 'rem' for instance

Reply:
:facepalm: you are so right Tom, thanks for that. 70% of width seems to be working for majority of devices to keep that date in Welsh in 1 line.